Formula/coreutils.rb

```diff
@@ -65,15 +71,22 @@ def install
       b2sum base32 chcon hostid md5sum nproc numfmt pinky ptx realpath runcon
       sha1sum sha224sum sha256sum sha384sum sha512sum shred shuf stdbuf tac timeout truncate
     ]
+    on_linux do
+      no_conflict += ["dir", "dircolors", "vdir"]
+    end
     no_conflict.each do |cmd|
       bin.install_symlink "g#{cmd}" => cmd
       man1.install_symlink "g#{cmd}.1" => "#{cmd}.1"
     end
   end
 
   def caveats
+    msg = "Commands also provided by macOS"
+    on_linux do
+      "All commands"
+    end
```

Comment:
I'm not convinced this block does what (I think that) you think it does...

Reply:
That was actually a typo. Fixed. Good catch.